docs/_build/html/.buildinfo

```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
```

Comment:
@jofsky the docs/_build directory should be added to .gitignore and not checked in to git.